pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -1794,3 +1811,71 @@ func isDeployDB(dbType api.DatabaseType) bool {
 	}
 	return false
 }
+
+func getCORSConfig(cors *api.CORSFiltersObject) {
+	if cors != nil {
+		//if something is missing we set mandatory defaults
+		AcFilters(cors)
```

Comment:
is there a real need to make function of every cors option?
I didn't see these funcs be called on other place than here, can you move everything to the same func and set the default values as needed?

Reply:
ok